djangocms_link/forms.py

```diff
@@ -1,34 +1,38 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import, print_function, unicode_literals
-
 from django.forms import ValidationError
 from django.forms.models import ModelForm
 from django.forms.widgets import Media
 from django.utils.translation import ugettext_lazy as _
 
 from djangocms_attributes_field.widgets import AttributesWidget
 
-from djangocms_link.models import Link
+from .models import Link
 
 
 class LinkForm(ModelForm):
     try:
         from djangocms_link.fields import PageSearchField
-        page_link = PageSearchField(label=_('Page'), required=False)
+        internal_link = PageSearchField(
+            label=_('Internal link'),
+            required=False,
+        )
     except ImportError:
         from cms.forms.fields import PageSelectFormField
-        page_link = PageSelectFormField(label=_('Page'), required=False)
+        internal_link = PageSelectFormField(
+            label=_('Internal link'),
+            required=False,
+        )
 
     def for_site(self, site):
```

Comment:
Instead of this, use something like https://docs.djangoproject.com/en/1.10/ref/contrib/admin/#django.contrib.admin.ModelAdmin.formfield_for_foreignkey